requirements.txt

```diff
@@ -6,4 +6,5 @@ jellyfish
 sphinx
 furo
 sphinx-copybutton
-twine
\ No newline at end of file
+twine
+scipy
```

Comment:
should also be added to dependencies in ``setup.py``

Reply:
yes

Reply:
done